test/integration_test.go

```diff
@@ -873,6 +874,39 @@ func TestTerragruntOutputFromRemoteState(t *testing.T) {
 
 	assert.True(t, (strings.Index(output, "app3 output") < strings.Index(output, "app1 output")) && (strings.Index(output, "app1 output") < strings.Index(output, "app2 output")))
 }
+
+func TestTerragruntOutputFromDependency(t *testing.T) {
+	// you cannot use t.Parallel() with t.Setenv
+
+	fixtureAppPath := filepath.Join(TEST_FIXTURE_OUTPUT_FROM_DEPENDENCY, "app")
+	fixtureDepPath := filepath.Join(TEST_FIXTURE_OUTPUT_FROM_DEPENDENCY, "dependency")
+
+	s3BucketName := fmt.Sprintf("terragrunt-test-bucket-%s", strings.ToLower(uniqueId()))
+	//defer deleteS3Bucket(t, TERRAFORM_REMOTE_STATE_S3_REGION, s3BucketName)
```

Comment:
I think this part should be enabled

Reply:
> I think this part should be enabled

Oh, I commented out when debugging and forgot to put it back. Thanks for catching this up. Fixed.